commercetools-sdk-base/src/main/java/io/sphere/sdk/queries/PagedResult.java

```diff
@@ -50,6 +50,20 @@
      }
 
     /**
+     * Calculates the page number of the result.
+     *
+     * @return the page number of the result.
+     */
+    default Long getPage() {
+
+        if (getCount() == 0) {
+            throw new UnsupportedOperationException("Can only be used if the count is known, and is greater than 0");
```

Comment:
I would prefer that we don't throw an exception here and instead just return `0L`. Otherwise a user of this method needs to add a `try/catch` block. Your solution is correct in a mathematical sense, but as a user of our SDK I don't like to catch unchecked exceptions.